src/generator.js

```diff
@@ -118,6 +141,22 @@ export const degenObject = <CustomType: string, CustomImport: string>(
 export const degenField = <CustomType: string, CustomImport: string>(
   fieldName: string,
   deserializer: DeserializerGenerator<CustomType, CustomImport>,
+): FieldDeserializer<CustomType, CustomImport> => {
+  const [deserializerFn, deps] = deserializer
+  return [fieldName, [() => {
+    // the `else {` is terminated in deObject during the join.
+    return `deField('${fieldName}',
+        (${deserializerFn()}),
+        json.${fieldName})`
+  },
+    mergeDeps(deps, { types: [], imports: ['deField'], hoists: [] }),
+  ]]
+}
+
+
+export const degenOptionalField = <CustomType: string, CustomImport: string>(
```

Comment:
It may be worth calling out in the documentation when one would use `degenOptionalField` over `degenField`, especially since the example doesn't use `degenOptionalField` at all.

Reply:
Ugh - it's vestigial. I'll make sure it's removed.

Reply:
Removed.